libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -135,7 +137,7 @@ import java.util.concurrent.ConcurrentSkipListSet
 import java.util.regex.Pattern
 import com.salesforce.androidsdk.auth.idp.IDPManager as DefaultIDPManager
 import com.salesforce.androidsdk.auth.idp.SPManager as DefaultSPManager
-import com.salesforce.androidsdk.security.interfaces.BiometricAuthenticationManager as DefaultBiometricAuthenticationManager
+import com.salesforce.androidsdk.security.interfaces.BiometricAuthenticationManager as BiometricAuthenticationManagerInterface
```

Comment:
I was just noticing this interface and internal implementation looks just like NativeLoginManager where we have the interface an an explicitly `internal` implementation.  `InternalBiometricAuthenticationManager`?

Of course, there's lot of ways to name interface/protocols and implementations 🤯